bin/s2nc.c

```diff
@@ -406,7 +404,7 @@ int main(int argc, char *const *argv)
             GUARD_EXIT(s2n_config_set_session_tickets_onoff(config, 1), "Error enabling session tickets");
         }
 
-        if (use_tls13 && S2N_IN_TEST) {
```

Comment:
I would argue we still need this flag, or customers can use TLS1.3 via s2nc and s2nd. Does Cloudfront need non-test TLS1.3 support in these clis?

Reply:
No, but I think he is just making it consistent.